### ISSUE TITLE
Two quests are unable to be completed

### ISSUE THREAD

**Issue by Timmy3921** [FTBTeam/FTB-Modpack-Issues#1297]
### Modpack

FTB Sky Odyssey

### Modpack version

1.4.0

### Describe the bug

Firstly, for the Black Hole Controller quest, it asks for the old Black Hole Controller, which is uncraftable. There is the Black Hole Contoller 2.0, but the quest won't accept that.

Secondly, for the No U Matter quest, you need solid UU matter from Tech Reborn and Industrialcraft 2 (not the liquid). However, there is no way to obtain solid UU matter from Industrialcraft 2. From old videos, you could put a UU matter cell in a Compressor, but that doesn't work anymore.

### Steps to reproduce

Just check the quest for black hole controller and the crafting recipe for UU matter (industrialcraft 2)

### Expected behaviour

I expected to be able to obtain UU matter and that the black hole controller quest would require the Black Hole Controller 2.0

**Comment by Raidobw2** [comment]
Thanks for reporting! I'll guide you through fixes, simply follow my directions.

I've tested those fixes and it should work, but let me know if they work for you too!

If you play on the FTB App, the modpack folder is accessed within Sky Odyssey's Overview page in your installed modpacks. Click "Settings" and scroll down to "Open Folder".

As well for the FTB App, after modifying those files, you'll also get a warning upon pressing play about modifications. You can ignore it and launch anyways. I know this message can be annoying when modifications are intended by the user, so I suggested to the FTB App team to not warn the user everytime about this.

## Fixing the black hole controller quest
1. Open your modpack folder
2. Locate the `config` folder and open it
3. Navigate to this path: `ftbquests/normal/chapters/` and then enter the `94f242d2` folder.
4. You'll see a file called `9bfe11e1.snbt`, right click it and open it with Notepad
5. Add `_reworked` at the end of the item, like on this picture:
![image](https://user-images.githubusercontent.com/17010632/187064442-3e1918f1-b400-4fc0-85cd-5e71550fdb2e.png)
6. Save the file, that's all for this one!

## Fixing UU Matter production
1. Open your modpack folder
2. Locate the `scripts` folder and open it
3. Right click the `RecipeChanges.zs` file and open it with Notepad
4. Shortly after the start of the file, add the following text (you can copy-paste it):
```
#Add recipe for Mass Fabricator
recipes.addShaped(<ic2:te:92>,
	[
		[<ore:dustGlowstone>, <ic2:crafting:2>, <ore:dustGlowstone>],
		[<ic2:resource:13>, <ic2:energy_crystal>.withTag({charge: 1000000.0}), <ic2:resource:13>],
		[<ore:dustGlowstone>, <ic2:crafting:2>, <ore:dustGlowstone>]
	]
);
```
5. It will look like this after you've pasted it:
![image](https://user-images.githubusercontent.com/17010632/187064739-7ef13862-3d03-480d-8e1d-4309fc1ed19a.png)
6. Save the file!
7. In-game, make the **Mass Fabricator**. Give it EU and some Scrap and it will make solid UU Matter. Use Transformer upgrades to be able to feed it more power, and make it go faster:

https://user-images.githubusercontent.com/17010632/187064782-ee259e99-e87b-4597-bf4b-2a7c8703a3e5.mp4

## For the next modpack developer that updates this
The scripts at `scripts/RecipeChanges.zs` had some lines which no longer work. We would have to adapt those lines. Or we can also not adapt them and just go with the Mass Fabricator instead. If so, gotta add some JEI info to solid IC2 UU Matter as well to state means of production though Mass Fab.

We could remove those lines simply:
![image](https://user-images.githubusercontent.com/17010632/187064575-13b64f45-4377-49b8-b67d-5c7368864fa9.png)
Basically the tech reborn compressor doesn't look like it's a thing anymore (after the 1.4.0 update, the mod updated I'm assuming it's that). There's an implosion compressor now. We could use the IC2 compressor, but IC2Tweaker isn't installed and the `config/ic2/XXXXX.ini` files are limited in terms of recipe NBT. Could also use the implosion compressor if we wanted, maybe it's easier. Idk, to be decided. https://docs.blamejared.com/1.12/en/Mods/Tech_Reborn/Implosion_Compressor

**Comment by TheonlyTazz** [comment]
I actually managed to add IC2 UU matter (Fluid) back to the IC2 Compressor 
![image](https://user-images.githubusercontent.com/3908847/187143999-711d0bd4-79c0-4ea2-9a50-aa73d14fc31d.png)
by adding
; UU-Matter cells
`Fluid:ic2uu_matter*1000 = ic2:misc_resource#matter`

to the ic2 config,  this accepts any container with 1000mb of liquid UU_Matter and outputs ic2:uu_matter
This can be troublesome as this will void containers that arent buckets. While this is not an issue for the techreborn cells, this is troublesome for stuff like Thermal tanks. IC2 cells dont even seem to work


Another alternative (as i do not want to use the Implosion Chamber multiblock) would be to use a fluid transposer like so
![image](https://user-images.githubusercontent.com/3908847/187175675-02d6b837-71ab-474c-92db-ce349deff8c3.png)

**Comment by Raidobw2** [comment]
> to the ic2 config, this accepts any container with 1000mb of liquid UU_Matter and outputs ic2:uu_matter

I see, I did not try it since I didn't think it worked like that. That's perfect then!

**Comment by Raidobw2** [comment]
Fixed in 1.4.1+ I believe. 1.4.1 is released and has the fix
